hail/src/main/scala/is/hail/expr/types/physical/PBaseStruct.scala

```diff
@@ -134,6 +134,8 @@ abstract class PBaseStruct extends PType {
     }
   }
 
+  def nMissing: Int
```

Comment:
seems no reason for this not to be on base struct, present in both PTuple and PStruct